GWAS/GWAS_QC.ipynb

```diff
@@ -416,9 +408,11 @@
     "    kin0 <- read.table(${_output[0]:r}, header=F, stringsAsFactor=F)\n",
     "    colnames(kin0) <- c(\"FID1\",\"ID1\",\"FID2\",\"ID2\",\"NSNP\",\"HETHET\",\"IBS0\",\"KINSHIP\")\n",
     "    if (${\"TRUE\" if maximize_unrelated else \"FALSE\"}) {\n",
+    "        # failIDs are the related IDs (pair-wise > threshold )\n",
     "        rel <- relatednessFilter(kin0, ${kinship}, \"ID1\", \"ID2\", \"KINSHIP\")$failIDs\n",
-    "        # FIXME: get the family ID for these rels so there are two columns FID and IID in the output\n",
-    "        dat = ...\n",
+    "        # get the family ID for these rels so there are two columns FID and IID in the output\n",
+    "        # will feed this file into PLINK -- keep/--remove for related/unrelated sample extraction\n",
+    "        dat<-cbind(rel,rel)\n",
```

Comment:
this is wrong. Family ID can be different from individual ID!!

Reply:
@haoyueshuai please try to query family ID out of kin0 based on individual ID and update the patch.

Reply:
@gaow when FID and IID are different, PLINK could not recognize the outlier file, and then nothing got filtered out. PLINK can only recognize when 2 columns are the same, both as IID. I just tested it again.

Reply:
@haoyueshuai i dont believe this is correct (do you believe it?) although it might be what is going on.  But just confiremd with @dianacornejo she has experience the same before. Could you try a different version of PLINK in case it is a bug?